app/locales/pt-BR/translations.js

```diff
@@ -75,10 +630,344 @@ export default {
+      savedMessage: 'O registro de medicação foi salvo.',
+      fulfilledTitle: 'Requisição de Medicação Preenchido'
+    }
+  },
+  appointments: {
+    currentScreenTitle: 'Lista de Compromisso',
+    editTitle: 'Editar Compromisso',
+    newTitle: 'Novo Compromisso',
+    sectionTitle: 'Compromissos',
+    thisWeek: 'Compromissos dessa Semana',
+    missed: 'Compromissos Esquecidos',
+    searchTitle: 'Pesquisar Compromissos',
+    todayTitle: 'Compromissos de Hoje',
+    messages: {
+      deleteAppointmentMessage: 'Você tem certeza que deseja excluir esse compromisso?',
+      endTimeLaterThanStart: 'Por favor, selecione um horário de término mais tarde do que a hora de início.'
+    },
+    buttons: {
+      newButton: '+ novo compromisso'
+    },
+    labels: {
+      selectedStartingDate: 'Mostrar Compromisso em ou após'
+    }
+  },
+  vitals: {
+    messages: {
+      delete: 'Você tem certeza que deseja excluir esses sinais vitais?'
+    },
+    labels: {
+      dateRecorded: 'Data de Gravação',
+      temperature: 'Temperatura',
+      weight: 'Peso',
+      height: 'Altura',
+      sbp: 'PAS',
+      dbp: 'PAD',
+      heartRate: 'Frequência Cardíaca',
+      respiratoryRate: 'Frequência Respiratória'
+    }
+  },
+  visits: {
+    titles: {
+      additionalDiagnoses: 'Diagnósticos Adicionais'
+    },
+    messages: {
+      delete: 'Você tem certeza que deseja excluir essa visita?'
+    },
+    buttons: {
+      newProcedure: 'Novo Procedimento',
+      newLab: 'Novo Laboratório',
+      newAppointment: 'Novo Compromisso',
+      addDiagnosis: 'Adicionar Diagnóstico',
+      newImaging: 'Nova Imagem',
+      newMedication: 'Nova Medicação',
+      newVitals: 'Novos Sinais Vitais'
+    },
+    labels: {
+      diagnosis: 'Diagnósticos',
+      procedure: 'Procedimentos',
+      authoredBy: 'Autorizado Por',
+      labs: 'Laboratório',
+      imaging: 'Imagem',
+      visitInformation: 'Imformação de Visita',
+      examiner: 'Examinador',
+      medication: 'Medicação',
+      status: 'Situação de Visita',
+      admittingDiagnosis: 'Diagnósticos de Admisão',
+      finalDiagnosis: 'Diagnósticos de Finais/Faturamento',
+      visitType: 'Tipo de Visita',
+      vitals: 'Sinais Vitais'
+    }
+  },
+  labs: {
+    sectionTitle: 'Laboratório',
+    requestsTitle: 'Requisições de Laboratório',
+    editTitle: 'Editar Requisição de Laboratório',
+    newTitle: 'Nova Requisição de Laboratório',
+    deleteTitle: 'Excluir Requisição',
+    completedTitle: 'Laboratórios Concluídos',
+    labels: {
+      labType: 'Tipo de Laboratório',
+      addNewVisit: '--Adicionar Nova Visita--'
+    },
+    messages: {
+      noItemsFound: 'Nenhum laboratório encontrado.',
+      createNewRecord: 'Criar novo registro?',
+      confirmDeletion: 'VOcê tem certeza que deseja excluir essa requisição de laboratório?',
+      noCompleted: 'Items concluídos não econtrados.'
+    },
+    buttons: {//AQUI
```

Comment:
@b0nn13 This line is causing an error because there needs to be a space after a line comment. 

Can you remove this comment or are you still working on this PR?